tests/middleware/tests.py

```diff
@@ -270,6 +270,42 @@ def test_prepend_www_append_slash_slashless_custom_urlconf(self):
         self.assertEqual(r.url,
             'http://www.testserver/customurlconf/slash/')
 
+    # ETag + If-Not-Modified support tests
+
+    @override_settings(USE_ETAGS=True)
+    def test_etag(self):
+        req = HttpRequest()
+        res = HttpResponse('content')
+        self.assertTrue(
+            CommonMiddleware().process_response(req, res).has_header('ETag'))
```

Comment:
While moving these you could also make the assertions a single line as we've relaxed our guidelines about line length when it helps readability.